crates/maili/src/lib.rs

```diff
@@ -21,7 +21,7 @@ pub use maili_rpc as rpc;
 
 #[cfg(feature = "common")]
 #[doc(inline)]
-pub use maili_common as common;
+pub use maili_consensus as common;
```

Comment:
I think we'll want to export this as `consensus` now instead of `common`.

```suggestion
#[cfg(feature = "consensus")]
#[doc(inline)]
pub use maili_consensus as consensus;
```